tests/conftest.py

```diff
@@ -503,6 +504,35 @@ def aarch64(body):
     assert mitigated, "SPECTREv2 not mitigated {}".format(body)
 
 
+def firecracker_id(fc):
+    """Render a nice ID for pytest parametrize."""
+    if isinstance(fc, FirecrackerArtifact):
+        return f"firecracker-{fc.version}"
+    return None
```

Comment:
```suggestion
def firecracker_id(fc):
    """Render a nice ID for pytest parametrize."""
    return f"firecracker-{fc.version}" if isinstance(fc, FirecrackerArtifact) else None
```
or possibly
```python
firecracker_id = lambda fc: f"firecracker-{fc.version}" if isinstance(fc, FirecrackerArtifact) else None
```
but I think the Python linter doesn't like lambdas.

Reply:
I think I will leave it as the longer version as I find it a tad more readable. Plus it would nicely extend to other classes in the future.

Reply:
I would prefer 3 lines over 1 in this case. Easier on the eyes.